test/include/kernels/DefaultMatPropConsumerKernel.h

```diff
@@ -24,3 +24,26 @@ class DefaultMatPropConsumerKernel : public DerivativeMaterialInterface<Kernel>
 
   const MaterialProperty<Real> & _prop;
 };
+
+/*
+
+#include "GenericKernel.h"
+#include "DerivativeMaterialInterface.h"
+
+template <bool is_ad>
+class DefaultMatPropConsumerKernelTempl : public DerivativeMaterialInterface<GenericKernel<is_ad>>
+{
+public:
+  static InputParameters validParams();
+
+  DefaultMatPropConsumerKernelTempl(const InputParameters & parameters);
+
+protected:
+  virtual GenericReal<is_ad> computeQpResidual() { return 0.0; };
+
+  const GenericMaterialProperty<Real, is_ad> & _prop;
+};
+
+typedef DefaultMatPropConsumerKernelTempl<false> DefaultMatPropConsumerKernel;
+typedef DefaultMatPropConsumerKernelTempl<true> ADDefaultMatPropConsumerKernel;
+ */
```

Comment:
Whatcha doin?

Reply:
Could you phrase your question a little more specific?

Reply:
Oh, I see a leftover commented piece of code. I was waiting for the GenericKernel PR. If that's in I can simplify the code.

Reply:
Got it. Do you want this in first?